requirements/requirements.txt

```diff
@@ -11,4 +11,4 @@ torch
 transformers
 ujson
 Werkzeug
-zmq
+pyzmq
```

Comment:
Thanks for this change @ccoulombe!  The failed formatting check is looking for the requirements list to be in alphabetical order.  Could you move this up to be below pydantic?

Reply:
@loadams ah, sure!

Reply:
Done!

Reply:
Thanks!